### ISSUE TITLE
[Accessibility] Role is not defined for stepUp and stepDown button

### ISSUE THREAD

**Issue by anmohanty123** [vlad-ignatov/react-numeric-input#133]
Go to http://vlad-ignatov.github.io/react-numeric-input/

Inspect the step Up button
![image](https://user-images.githubusercontent.com/104103001/172489704-952e4fe0-498e-4d56-a4dc-48cdfddb7bc0.png)

due to this issue screen reader does not announce any information when focus goes on spin button.
